src/components/canrawview/canrawview_p.h

```diff
@@ -90,17 +121,41 @@ class CanRawViewPrivate : public QWidget {
     CanRawView* q_ptr;
     int prevIndex = 0;
     int rowID = 0;
+    QStringList columnsOrder;
+
+    void writeColumnsOrder(QJsonObject& json) const
+    {
+        uint ii = 0;
```

Comment:
`bool QTableView::isColumnHidden(int column) const` thus I will change this to `int index = 0;`.